### PR TITLE
Cscfairmeta 526 qvain light ui changes part 2

### PR DESCRIPTION
**Pull request by ismotee**
Bug fixes:
- corrected typo from finnish translation of consent
- organizations translations did not change when changing language
- data catalog options did not change when changing language
- validation error on description did not change when changing language

Ui improvements:
- Keywords now being added when the user presses enter or ","
- hide "needs Fairdata permission" acces type when rems is disabled
- file buttons renamed: cancel => clear fields, Save => Add external file

### CODE REVIEW

File: etsin_finder/frontend/js/components/qvain/files/dataCatalog.jsx [hunk 1]
```diff
@@ -36,6 +36,16 @@ class DataCatalog extends Component {
...
 
+  updateOptions = () => {
+    options = [
+      { value: DataCatalogIdentifiers.IDA, label: translate('qvain.files.dataCatalog.ida') },
+      { value: DataCatalogIdentifiers.ATT, label: translate('qvain.files.dataCatalog.att') }
+    ]
+    pasOptions = [
+      { value: DataCatalogIdentifiers.PAS, label: translate('qvain.files.dataCatalog.pas') }
+    ]
+        }
```
Comment (tahme) [inline etsin_finder/frontend/js/components/qvain/files/dataCatalog.jsx:47]: too much indent for }

File: etsin_finder/frontend/js/components/qvain/licenses/accessType.jsx [hunk 2]
```diff
@@ -20,59 +20,69 @@ export class AccessType extends Component {
...
+          }))
+
+          const user = this.props.Stores.Auth.user
+
+          if (!user.isUsingRems) {
+            // Removing one option from the list "Vaatii luvan hakemista Fairdata palvelusta".
+            // Assuming the option to be the last in the list.
+            // Other ways to do this is to check it from url or from description.
+            // Any of these ways is somewhat bad.
+            // It is recommended that this removal is done in the server not in the client.
```
Comment (tahme) [inline etsin_finder/frontend/js/components/qvain/licenses/accessType.jsx:55]: use the uri from `constants.js` for filtering

File: etsin_finder/frontend/js/components/qvain/files/dataCatalog.jsx (hunk 1 above)
Comment (ismotee) [reply etsin_finder/frontend/js/components/qvain/files/dataCatalog.jsx]: fixed

File: etsin_finder/frontend/js/components/qvain/licenses/accessType.jsx (hunk 2 above)
Comment (ismotee) [reply etsin_finder/frontend/js/components/qvain/licenses/accessType.jsx]: fixed as suggested

**Pull request merged**